etc/docker/hangar-test-maven/include/project-1/test-1.bats

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bats
+
+@test "Compiled target jar exists" {
+  result="$(ls -l ./project-1/target/hangar-test-0.0.1-SNAPSHOT.jar | wc -l)"
+  [ "$result" -eq 1 ]
```

Comment:
Turns out that doesn't work because it's the exit code of test I need, not the result. Still, good catch.